db/migrate/20181109163137_create_webhook.rb

```diff
@@ -0,0 +1,17 @@
+class CreateWebhook < ActiveRecord::Migration[5.1]
+  def change
+    create_table :webhooks do |t|
+      t.bigint :tenant_id
+      t.string :name
+      t.string :url
+      t.boolean :verify_ssl
+      t.string :secret
+      t.string :authentication
+      t.string :userid
+      t.string :password
+      t.string :token
```

Comment:
If you are storing secrets in this table, we will need the PasswordMixin in the model.  At a minimum I see secret and password, and maybe token.  I'm also seeing authentication though I don't know what that means.